Automattic/sensei
# Add a client-rendered block to reproduce the sensei_user_message shortcode

Add the Sensei Messages block.

### Testing Instructions

- Send some messages on a Sensei Course or Lesson page.
- Create a page in the block editor.
- In the "Sensei LMS" section, add the "Sensei LMS Messages" block.
- You should see your message threads show up in the editor.
- Save and view the page. The messages should also show up on the frontend.
